applications/nrf_desktop/src/modules/watchdog.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/watchdog.h>
+
+#define MODULE watchdog
+#include "module_state_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_WATCHDOG_LOG_LEVEL);
+
+
+#if defined(CONFIG_WDT_0_NAME)
+#define WATCHDOG_DEV_NAME CONFIG_WDT_0_NAME
```

Comment:
Let's leave only the kconfig name from Zephyr. Watchdog needs to be enabled there anyway.

Reply:
ok